tsup.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig, Options } from "tsup";
+import path from "path";
+
+const baseOptions: Options = {
+  clean: true,
+  dts: true,
+  entry: ["index.?s"], // while we have a mix of TS and JS packages
+  minify: false,
+  skipNodeModulesBundle: true,
+  sourcemap: true,
+  target: "es2017",
+  tsconfig: "tsconfig.json",
+  keepNames: true,
+  // treeshake: true, causes "chunk.default" warning, breaks CJS exports?
+  cjsInterop: false, // putting this to true will break backwards compatability
```

Comment:
Currently, when importing `require("@turf/foo")`, you will get back an object containing a property called `default`, which is the default export. If the property above to false, this will not be the case and the default export will be the actual export if there is only 1 export. However, this is a breaking change, so I had to but this to false.